File: aat-awt/src/test/java/ch/bailu/aat_awt/TestTest.java
package ch.bailu.aat_awt;


import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class TestTest {

    boolean initValue = false;

    @BeforeEach
    public void init() {
        initValue = true;
    }

    @Test
    public void testTest() {
        assertEquals(true, initValue);
    }

    @AfterEach
    public void reset() {
        initValue = false;
    }
}

File: aat-android/src/test/kotlin/ch/bailu/aat/TestTest.kt
package ch.bailu.aat

import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

class TestTest {
    var initValue = false
    @BeforeEach
    fun init() {
        initValue = true
    }

    @Test
    fun testTest() {
        Assertions.assertEquals(true, initValue, "This test failed")
    }

    @AfterEach
    fun reset() {
        initValue = false
    }
}
